resources/assets/js/components/ApplicationReview/helpers.ts

```diff
@@ -15,6 +15,16 @@ export function isScreenedOut(application: Application): boolean {
     : false; // non-reviewed applications have not been screened-out yet
 }
 
+/**
+ * Returns true if application has been screened in.
+ */
+export function isStillIn(application: Application): boolean {
+  return application.application_review &&
+    application.application_review.review_status
+    ? application.application_review.review_status.name === "still_in"
+    : false; // non-reviewed applications have not been screened-out yet
+}
```

Comment:
Check this out! [Typescript optional chaining.](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining)

You can replace all that with `application?.application_review?.review_status?.name === "still_in"`. If any of the steps is null or undefined, it will cut evaluation short and return `undefined`. Which will, of course not, equal "still_in".

Reply:
Awesome thanks!